exporter/otlp-grpc/lib/opentelemetry/exporter/otlp/grpc.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry/exporter/otlp/grpc/exporter'
+require 'opentelemetry/exporter/otlp/grpc/version'
+
+# OpenTelemetry is an open source observability framework, providing a
+# general-purpose API, SDK, and related tools required for the instrumentation
+# of cloud-native software, frameworks, and libraries.
+#
+# The OpenTelemetry module provides global accessors for telemetry objects.
+# See the documentation for the `opentelemetry-api` gem for details.
+module OpenTelemetry
```

Comment:
Do we normally do things this way, or do we:
```ruby
module OpenTelemetry
  module Exporter
    module OTLP
      module GRPC
      end
    end
  end
end

require 'opentelemetry/exporter/otlp/grpc/exporter'
require 'opentelemetry/exporter/otlp/grpc/version'
```

Reply:
Will update.